scripts/basic-workflow-tests.sh

```diff
@@ -296,8 +297,60 @@ test_train() {
 
         ilab model train "${TRAIN_ARGS[@]}"
     fi
+}
+
+test_phased_train() {
+    task Train the model with LAB multi-phase strategy.
+
+    # invariant requirement
+    if [ "$TRAIN_LIBRARY" -eq 0 ]; then
+        echo "Cannot run phased training without full train library. Exiting."
+        return
+    fi
+
+    # need to do piece-wise because 'Declare and assign separately to avoid masking return values'
```

Comment:
I don't understand the comment. Could you please re-word it somehow?

Reply:
updated